packages/block-editor/src/components/spacing-sizes-control/README.md

```diff
@@ -0,0 +1,99 @@
+# Spacing Sizes Control
+
+The SpacingSizesControl component provides a user interface for controlling spacing values in blocks. It supports single, axial, and separated input controls for different spacing configurations.
+
+## Description
+
+The SpacingSizesControl component is a flexible control that allows users to modify spacing values for different sides of a block. It supports three viewing modes:
+
+1. Single: Control one side at a time
+2. Axial: Control horizontal and vertical sides together
+3. Custom: Control each side separately
+
+![Spacing Sizes Control](https://i.postimg.cc/3RkzzfL6/Screenshot-2025-01-10-at-8-07-55-AM.png)
+
+## Usage
+
+```jsx
+import { SpacingSizesControl } from '@wordpress/block-editor';
+import { useState } from '@wordpress/element';
+
+function Example() {
+	const [ sides, setSides ] = useState( {
+		top: '0px',
+		right: '0px',
+		bottom: '0px',
+		left: '0px',
+	} );
+
+	return (
+		<SpacingSizesControl
+			values={ sides }
+			onChange={ setSides }
+			label="Sides"
+		/>
+	);
+}
+```
+
+## Props
+
+### `inputProps`
+
+-   Type: `Object`
+-   Required: No
+-   Description: Additional props to pass to the input controls.
+
+### `label`
+
+-   Type: `String`
+-   Required: Yes
+-   Description: Label for the control (e.g., "Height").
```

Comment:
I think the example `(e.g., "Height")` is not a great one. I don’t think we need such an example anyway so I’d just remove that bit.